### ISSUE TITLE
volcanoly annotation not fully working?

### ISSUE THREAD

**Issue by sert23** [sahirbhatnagar/manhattanly#22]
Hi great package, I really enjoyed using it.
When I try to make a volcano plot using volcanoly function, I can't get the annotation to show on non-significant genes. That's not how it behaves in the example. For instance, you can run the tutorial example with the latest github version:

`volcanoly(HapMap, snp = "SNP", gene = "GENE")`

the SNP and GENE annotation only shows for the significant ones. 
I have tried a couple of workarounds but it didn't work so far.

**Comment by alouyakis** [comment]
I'm having the same problem. Is there a solution to this yet?

**Comment by sert23** [comment]
I solved it by forking the code and rewriting this function. It lost most of the functionality but I kept the features that I was interested in.

**Comment by alouyakis** [comment]
In the volcanoly function, I set highlight to the gene names in the input df from the volcanor object and highlight_color to "black". I lose the different color for everything outside my cutoff lines, but everything is labeled now.

**Comment by sert23** [comment]
can you maybe post an image or the html so I can understand a bit better what's wrong?
here's what I did a few months ago:
https://github.com/sert23/conDE/blob/659548a75600c2555f5875784fda4ff4d4a12c50/DEtools/Rplots/volcano.R

first lines (until 248) is rewriting of volcanoly function (I named it volcanoly2).
hope this helps!

**Comment by alouyakis** [comment]
I put it all up here:
https://alouyakis.github.io/volcano_fix/
You can ignore the ggplot volcano plots at the end.

**Comment by sahirbhatnagar** [comment]
Sorry for the delay. This should now be fixed in the newest release:

``` r
library(manhattanly)
#> See example usage at http://sahirbhatnagar.com/manhattanly/
data("HapMap")
p <- volcanoly(HapMap, snp = "SNP", gene = "GENE")
```

<sup>Created on 2021-04-20 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>
<details>
<summary>
Session info
</summary>

``` r
devtools::session_info()
#> ─ Session info ───────────────────────────────────────────────────────────────
#>  setting  value                       
#>  version  R version 4.0.2 (2020-06-22)
#>  os       Pop!_OS 20.10               
#>  system   x86_64, linux-gnu           
#>  ui       X11                         
#>  language en_US:en                    
#>  collate  en_US.UTF-8                 
#>  ctype    en_US.UTF-8                 
#>  tz       America/Toronto             
#>  date     2021-04-20                  
#> 
#> ─ Packages ───────────────────────────────────────────────────────────────────
#>  package     * version date       lib source                            
#>  assertthat    0.2.1   2019-03-21 [1] CRAN (R 4.0.2)                    
#>  blob          1.2.1   2020-01-20 [1] CRAN (R 4.0.2)                    
#>  callr         3.6.0   2021-03-28 [1] CRAN (R 4.0.2)                    
#>  cli           2.4.0   2021-04-05 [1] CRAN (R 4.0.2)                    
#>  colorspace    2.0-0   2020-11-11 [1] CRAN (R 4.0.2)                    
#>  crayon        1.4.1   2021-02-08 [1] CRAN (R 4.0.2)                    
#>  crosstalk     1.1.1   2021-01-12 [1] CRAN (R 4.0.2)                    
#>  data.table    1.14.0  2021-02-21 [1] CRAN (R 4.0.2)                    
#>  DBI           1.1.0   2019-12-15 [1] CRAN (R 4.0.2)                    
#>  desc          1.3.0   2021-03-05 [1] CRAN (R 4.0.2)                    
#>  devtools      2.3.2   2020-09-18 [1] CRAN (R 4.0.2)                    
#>  digest        0.6.27  2020-10-24 [1] CRAN (R 4.0.2)                    
#>  dplyr         1.0.5   2021-03-05 [1] CRAN (R 4.0.2)                    
#>  ellipsis      0.3.1   2020-05-15 [1] CRAN (R 4.0.2)                    
#>  evaluate      0.14    2019-05-28 [1] CRAN (R 4.0.2)                    
#>  fansi         0.4.2   2021-01-15 [1] CRAN (R 4.0.2)                    
#>  fs            1.5.0   2020-07-31 [1] CRAN (R 4.0.2)                    
#>  generics      0.1.0   2020-10-31 [1] CRAN (R 4.0.2)                    
#>  ggplot2       3.3.3   2020-12-30 [1] CRAN (R 4.0.2)                    
#>  glue          1.4.2   2020-08-27 [1] CRAN (R 4.0.2)                    
#>  gtable        0.3.0   2019-03-25 [1] CRAN (R 4.0.2)                    
#>  highr         0.8     2019-03-20 [1] CRAN (R 4.0.2)                    
#>  htmltools     0.5.1.1 2021-01-22 [1] CRAN (R 4.0.2)                    
#>  htmlwidgets   1.5.3   2020-12-10 [1] CRAN (R 4.0.2)                    
#>  httr          1.4.2   2020-07-20 [1] CRAN (R 4.0.2)                    
#>  jsonlite      1.7.2   2020-12-09 [1] CRAN (R 4.0.2)                    
#>  knitr         1.31    2021-01-27 [1] CRAN (R 4.0.2)                    
#>  labeling      0.4.2   2020-10-20 [1] CRAN (R 4.0.2)                    
#>  lazyeval      0.2.2   2019-03-15 [1] CRAN (R 4.0.2)                    
#>  lifecycle     1.0.0   2021-02-15 [1] CRAN (R 4.0.2)                    
#>  magrittr      2.0.1   2020-11-17 [1] CRAN (R 4.0.2)                    
#>  manhattanly * 0.2.0   2016-11-16 [1] CRAN (R 4.0.2)                    
#>  memoise       1.1.0   2017-04-21 [1] CRAN (R 4.0.2)                    
#>  munsell       0.5.0   2018-06-12 [1] CRAN (R 4.0.2)                    
#>  pillar        1.6.0   2021-04-13 [1] CRAN (R 4.0.2)                    
#>  pkgbuild      1.2.0   2020-12-15 [1] CRAN (R 4.0.2)                    
#>  pkgconfig     2.0.3   2019-09-22 [1] CRAN (R 4.0.2)                    
#>  pkgload       1.2.1   2021-04-06 [1] CRAN (R 4.0.2)                    
#>  plotly        4.9.3   2021-01-10 [1] CRAN (R 4.0.2)                    
#>  prettyunits   1.1.1   2020-01-24 [1] CRAN (R 4.0.2)                    
#>  processx      3.5.1   2021-04-04 [1] CRAN (R 4.0.2)                    
#>  ps            1.6.0   2021-02-28 [1] CRAN (R 4.0.2)                    
#>  purrr         0.3.4   2020-04-17 [1] CRAN (R 4.0.2)                    
#>  R6            2.5.0   2020-10-28 [1] CRAN (R 4.0.2)                    
#>  remotes       2.2.0   2020-07-21 [1] CRAN (R 4.0.2)                    
#>  rlang         0.4.10  2020-12-30 [1] CRAN (R 4.0.2)                    
#>  rmarkdown     2.7.7   2021-04-08 [1] Github (rstudio/rmarkdown@dcc03e5)
#>  rprojroot     2.0.2   2020-11-15 [1] CRAN (R 4.0.2)                    
#>  scales        1.1.1   2020-05-11 [1] CRAN (R 4.0.2)                    
#>  sessioninfo   1.1.1   2018-11-05 [1] CRAN (R 4.0.2)                    
#>  stringi       1.5.3   2020-09-09 [1] CRAN (R 4.0.2)                    
#>  stringr       1.4.0   2019-02-10 [1] CRAN (R 4.0.2)                    
#>  testthat      3.0.2   2021-02-14 [1] CRAN (R 4.0.2)                    
#>  tibble        3.1.1   2021-04-18 [1] CRAN (R 4.0.2)                    
#>  tidyr         1.1.3   2021-03-03 [1] CRAN (R 4.0.2)                    
#>  tidyselect    1.1.0   2020-05-11 [1] CRAN (R 4.0.2)                    
#>  usethis       2.0.1   2021-02-10 [1] CRAN (R 4.0.2)                    
#>  utf8          1.2.1   2021-03-12 [1] CRAN (R 4.0.2)                    
#>  vctrs         0.3.7   2021-03-29 [1] CRAN (R 4.0.2)                    
#>  viridisLite   0.4.0   2021-04-13 [1] CRAN (R 4.0.2)                    
#>  withr         2.4.2   2021-04-18 [1] CRAN (R 4.0.2)                    
#>  xfun          0.22    2021-03-11 [1] CRAN (R 4.0.2)                    
#>  yaml          2.2.1   2020-02-01 [1] CRAN (R 4.0.2)                    
#> 
#> [1] /home/sahir/R/x86_64-pc-linux-gnu-library/4.0
#> [2] /usr/local/lib/R/site-library
#> [3] /usr/lib/R/site-library
#> [4] /usr/lib/R/library
```

</details>

See the following link for resulting interactive plot:
https://plotly.com/~sahirbhatnagar/149/

**Comment by sahirbhatnagar** [comment]
Fixed with #24
